clusterloader2/testing/load/experimental/overrides/enable_configmaps.yaml

```diff
@@ -1 +0,0 @@
-ENABLE_CONFIGMAPS: true
```

Comment:
We cannot delete these files yet as they are being listed in job definitions. If we delete them the test jobs will start failing
We need 3 separate PRs, in the following order
1. Change the defaults to true
2. Change job definitions to not use override files (in test-infra repo)
3. Delete override files

Reply:
@mm4tt  I have reverted the file deletions. Should I delete the test-infra repo PR, or can I just I put a comment in the test-infra repo PR that that can only be merged after this PR is merged? Thanks Mani.

Reply:
I have closed the  test-infra PR.